src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -256,9 +255,7 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundIsPatternExpressionOperation((BoundIsPatternExpression)boundNode);
```

Comment:
Filed https://github.com/dotnet/roslyn/issues/21723 to track issues with IsLifted.